Speedmod/Movies/BestOf.yml

```diff
@@ -55,7 +50,8 @@ collections:
     summary: Popular Movies across the internet
   Top Rated Global:
```

Comment:
This collection does not look fixed to me, review the indentation of the different builders again